config/initializers/application_metadata.rb

```diff
@@ -0,0 +1,15 @@
+APPLICATION_METADATA = {
+  slug: APP_SLUG,
+  content_id: "69af22e0-da49-4810-9ee4-22b4666ac627",
+  title: "Licence Finder",
+  description: "Find out which licences you might need for your activity or business.",
+  section: "business",
+  paths: [],
+  prefixes: ["/#{APP_SLUG}"],
+  indexable_content: <<-HEREDOC.gsub(/\s+/, " ").strip,
```

Comment:
`strip_heredoc` doesn't quite do what we need here - it doesn't take care of newlines so you'd need the gsub anyway.